### PR TITLE
Add index file, update scripts to warn if service is already running …

### PR DESCRIPTION
**Pull request by khansaad**
…or stopped

Signed-off-by: Saad Khan <saakhan@redhat.com>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<html>
+<head>
+    <title>HPO</title>
```
Comment (dinogun) [inline index.html:3]: This needs to be `Hyper Parameter Optimization as a Service`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    
+    <p style="font-size:18px; font-family: Arial, Helvetica, sans-serif;"> HPO as a service helps the user find the optimal configuration values for any model. </p>
+    <h2 style="font-family: Arial, Helvetica, sans-serif;">Goal:</h2>
+    <p style="font-size:18px; font-family: Arial, Helvetica, sans-serif;">
+        Provide Hyperparameter Optimization(HPO) as a service to choose the optimal hyperparameters provided by the user for any model. 
+        Primary use case is to optimize the performance of an environment by tuning below layers independently or together.
+            * Container : resource usage like cpu and memory requests /limits.
+            * Runtime: Ex : hotspot, openj9, nodeJS , etc.
+            * Application stack:  Ex: Quarkus , Liberty, postgres etc.
+            * OS layer.
```
Comment (dinogun) [inline index.html:16]: Please remove this as this is not needed

File: index.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    </p>
+ 
+    <h2 style="font-family: Arial, Helvetica, sans-serif;"> Intended Users: </h2>
+    <p style="font-size:18px; font-family: Arial, Helvetica, sans-serif;">
+        To improve the performance of the runtime / application server we would have lots of tunables to tune and those will vary with the application and hardware we are running on. Manual tuning the multiple parameters with a large range is so cumbersome, and is hard to come up with an optimal configuration.
```
Comment (dinogun) [inline index.html:21]: This needs to be
```
Any model that has tens of parameters that need to be tuned, or the parameters operate on a wide range, or it is not clear which parameters have an impact on the performance objective are ideal candidates to use the HPOaaS.
```

File: index.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    </p>
+
+    <h2 style="font-family: Arial, Helvetica, sans-serif;">Here's a link which contains <a href="https://github.com/kruize/hpo/blob/main/README.md">API details</a></h2>
+</body>
+</html>
```
Comment (dinogun) [inline index.html:26]: Add a EoF here

**Review by dinogun — CHANGES_REQUESTED** [review]
The index.html needs to be packaged as part of the Dockerfile for this to work inside docker.

File: index.html (hunk 1 above)
Comment (khansaad) [reply index.html]: Done

File: index.html (hunk 2 above)
Comment (khansaad) [reply index.html]: Done

File: index.html (hunk 3 above)
Comment (khansaad) [reply index.html]: Done

File: index.html (hunk 4 above)
Comment (khansaad) [reply index.html]: Done

File: src/service.py [hunk 5]
```diff
@@ -85,11 +85,21 @@ def do_GET(self):
...
+            elif (not query):
+                data = self.getHomeScreen()
+                self._set_response(200, data)
             else:
                 self._set_response(404, "-1")
         else:
             self._set_response(403, "-1")
 
+    def getHomeScreen(self):
+        fin = open('index.html')
```
[the remark this answers is not in the document]
Comment (khansaad) [reply src/service.py]: Done

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**